tests/python/test_charge_deposition.py

```diff
@@ -63,7 +63,7 @@ def test_charge_deposition(save_png=True):
         bx = mfi.validbox()
         rbx = amrex.RealBox(bx, dr, gm.ProbLo())
 
-        arr = rho.array(mfi)
+        arr = rho.array(mfi)  # TODO: explicit device-to-host memory copy
```

Comment:
This is easy to do with cupy, but here I would potentially rely purely on AMReX for now if we can copy the whole MultiFab before the loop (or each box at this location).

Reply:
@WeiqunZhang commented:
```

Reply:
pyAMReX: https://github.com/AMReX-Codes/pyamrex/pull/123